src/app/components/champion/ChampionView.tsx

```diff
@@ -28,34 +29,55 @@ export const ChampionView: FC<ChampionProps> = ({champion, callback, championsLi
     const useStyles = makeStyles((theme: Theme) => ({
         icon: {
             verticalAlign: 'bottom',
-            height: 25,
-            width: 25,
+            height: 30,
+            width: 30,
+            color: theme.palette.secondary.main
         },
-        slider: {
-            paddingTop: '14%'
-        },
-        statBox: {
-            backgroundColor: theme.palette.primary.main
+        componentBox: {
+            backgroundColor: theme.palette.primary.main,
+            marginTop: '35px'
         },
         avatarSearch: {
-            height: '22%',
-            width: '22%',
-            marginRight:'5%'
-        }
+            height: '50px',
+            width: '50px',
+            marginRight: '5%'
+        },
+        avatarDisplay: {
+            height: '75px',
+            width: '75px',
+            marginRight: '20px'
+        },
+        itemTextSizePrimary: {
+            fontSize:'2em'
+        },
+        itemTextSizeSecondary: {
+            fontSize:'1.3em'
+        },
+        abilityIcon: {
+            marginRight: '3.6%'
+        },
     }));
 
     const classes = useStyles();
 
+    const AbilityTooltip = withStyles((theme: Theme) => ({
+        tooltip: {
+            backgroundColor: theme.palette.primary.main,
+            fontSize: 14
+        }
+    }))(Tooltip)
+
     return (
-        <List component="nav" aria-label="secondary mailbox folders" style={{width: '18%', margin:'auto'}}>
+        <List component="nav" aria-label="secondary mailbox folders">
             <Autocomplete
                 id="champion-box-complete"
                 options={championsList}
                 getOptionLabel={(option) => option.name}
                 onChange={callback}
                 renderOption={(option) => (
                     <React.Fragment>
-                        <Avatar className={classes.avatarSearch} src={"http://ddragon.leagueoflegends.com/cdn/11.6.1/img/champion/"+option.name+".png"}/>
+                        <Avatar className={classes.avatarSearch}
+                                src={"https://ddragon.leagueoflegends.com/cdn/11.6.1/img/champion/" + option.name + ".png"}/>
```

Comment:
Hmmmm a configuration for ddragon urls would be really nice. Especially because you have version in your url.
Simple Config : Create a config.ts and add an object config.

```
const statCheckConfig = {
    api : {
        url: "https://ddragon.leagueoflegends.com/",
        version: "11.6.1"
    }
}
```

You can also extract it in a method to compute this url. Avoid too much computing in your components

Reply:
This would be useful especially because I plan to also request the current game version to DataDragon, so it would end up being cleaner. I will deal with it in my API calls ticket(s).

Reply:
Will be done in this ticket : https://trello.com/c/RaJdspIO/26-5-15-use-a-reducer-to-normalize-the-axios-calls-and-add-an-error-message